tests/x11/lxqt_reconfigure_openqa.pm

```diff
@@ -20,6 +21,10 @@ sub run() {
 
     set_var("DESKTOP", "lxqt");
 
+    # LXQt uses lightdm as window manager, which has the user preselected
+    set_var('DISPLAYMANAGER',    'lightdm');
```

Comment:
do you mind add an comment out here to explain why set it to 0? I knew it because it sets to 1 when desktop is minimalx in main.pm, just the reference for anyone else :)